mobile/src/components/courses-management/courses-management.tsx

```diff
@@ -37,9 +41,23 @@ const CoursesManagement: FC = () => {
     navigation.navigate(AppScreenName.EDIT_COURSE);
   };
 
+  const handleTitlePress = (item: CourseGetResponseDto): void => {
+    dispatch(coursesActions.getCourse(item));
+    navigation.navigate(AppScreenName.COURSE);
+  };
+
+  navigation.setOptions({
+    title: <CoursesManagementTitle />,
```

Comment:
Let's just reduce the title 
```suggestion
    title: 'Courses',
```